js-exercises/user-check-password/user.js

```diff
@@ -0,0 +1,19 @@
+
+// (*)
+// Follow the instructions and fill in the blank sections.
+function User(username, password) {
+  // set a username and password property on the user object that is created\
+  this.username = username
+  this.password = password
+}
```

Comment:
Good that you've used prototypes to solve this. But, It would be nice if you use Object.create function with shared property/prototype as parameter and return the object in User function. It enables User function work without new keyword.